docs/DISTROS.md

```diff
@@ -0,0 +1,66 @@
+# curl distros
+
+Lots of organizations distribute curl packages to end users. This is a
+collection of pointers to where to learn more about curl on and with each
+distro.
+
+## AlmaLinux
+
+- curl: [URL]
+- curl issues: [URL]
+- curl patches: [URL]
+
+## Arch Linux
+
+- curl: https://archlinux.org/packages/core/x86_64/curl/
+- curl issues: https://gitlab.archlinux.org/archlinux/packaging/packages/curl/-/issues
+- curl security: https://security.archlinux.org/package/curl
+- curl wiki: https://wiki.archlinux.org/title/CURL
+
+## Debian
+
+- curl: https://tracker.debian.org/pkg/curl
+- curl issues: https://bugs.debian.org/cgi-bin/pkgreport.cgi?pkg=curl
+- curl patches: https://udd.debian.org/patches.cgi?src=curl
+
+## Fedora
+
+- curl: https://src.fedoraproject.org/rpms/curl
+- curl issues: [bugzilla](https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&classification=Fedora&product=Fedora&product=Fedora%20EPEL&component=curl)
+- curl patches: [list of patches in package git](https://src.fedoraproject.org/rpms/curl/tree/rawhide)
+
+## Homebrew
+
+- curl: https://formulae.brew.sh/formula/curl
+- curl issues: [URL]
+- curl patches: [URL]
```

Comment:
Really Homebrew's policy is that all patches should be submitted upstream unless it is very specific to Homebrew's way of packaging software. The issues could also just point to the generic Homebrew issue tracker. There is no specific curl issue tracker in Homebrew.

```suggestion
- curl issues: https://github.com/curl/curl/issues
- curl patches: https://github.com/curl/curl/pull
```